packages/vue/package.json

```diff
@@ -48,25 +50,29 @@
   "devDependencies": {
     "@babel/core": "^7.23.2",
     "@babel/preset-env": "^7.23.2",
+    "@rollup/plugin-alias": "^5.1.1",
     "@rollup/plugin-dynamic-import-vars": "^1.4.0",
+    "@rollup/plugin-node-resolve": "^16.0.0",
     "@rushstack/eslint-patch": "^1.5.1",
     "@testing-library/vue": "^6.6.0",
     "@types/node": "18.8.0",
-    "@vitejs/plugin-vue-jsx": "^3.1.0",
     "@vue/babel-plugin-jsx": "^1.2.2",
+    "@vue/compiler-sfc": "^3.3.4",
     "@vue/eslint-config-prettier": "^8.0.0",
     "@vue/eslint-config-typescript": "^12.0.0",
     "@vue/test-utils": "^2.4.1",
     "@vue/vue3-jest": "^29.2.6",
     "babel-jest": "^29.7.0",
     "eslint-plugin-vue": "^9.17.0",
-    "rollup-plugin-typescript2": "^0.31.2",
+    "rollup-plugin-postcss": "^4.0.2",
+    "rollup-plugin-typescript2": "^0.36.0",
+    "rollup-plugin-vue": "^6.0.0",
     "unplugin-vue-components": "^0.25.0",
-    "vite-jest": "^0.1.4"
+    "vue": "^3.3.4"
   },
   "peerDependencies": {
     "@aws-amplify/core": "*",
-    "aws-amplify": "^6.9.0",
+    "aws-amplify": "*",
```

Comment:
What is the reasoning behind this change? Not just any version of `aws-amplify` is compatible with this package right?